src/plugins/mer/merhardwaredevicewizardpages.cpp

```diff
@@ -148,6 +152,27 @@ void MerHardwareDeviceWizardSelectionPage::handleTestConnectionClicked()
         goto end;
     }
 
+    m_ui->connectionTestLabel->setText(tr("Detecting SDK tools..."));
```

Comment:
"SDK Client Tools" doesn't really say anything to the users. How about "SDK Device Tools", which would indicate that these are tools that are needed on the device in order for it to work with the SDK.

Reply:
I can change it to whatever feels correct. SDK Device Tools seems like a good one and there is not too much space for long descriptions.

Reply:
I agree with the space constraints but I need to stress again than we do not really need a term, and if we do not need it, I would prefer not introducing it - it's not for for free.

Reply:
So do you want it to be changed or not? I didn't quite understand.

We could also have a more descriptive text above the test button if needed. Something like "You may be prompted to install software packages required for SDK operation such as deploying packages on device" and then just call it SDK tools or SDK packages elsewhere.

Reply:
Sorry :) I prefer saying "{Detecting|Installing|Failed to install|...} software packages required for proper SDK operation...", including additional information in case of failure, and adjust the UI to remove the space constraint - moving the label below the button and using multiple lines?